index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <link rel="stylesheet" href="style.css">
+
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+        <header>
+        <section class="toolbar">
```

Comment:
- [ ] Just like the first reviewer mentioned, the `section` tag should not be inside a `header` because the `header` tag is a section on its own. I would advise you to change this so that it becomes semantic. 👍 

#### Hint:

```HTML
<header>
  <a href="#">logo</a>
  <nav>hamburger button</nav>
</header>
```